bench/bench.ml

```diff
@@ -1,23 +1,13 @@
-let crypto_random_bytes n =
+let read_urandom n =
   let ic = Pervasives.open_in_bin "/dev/urandom" in
   let s = Pervasives.really_input_string ic n in
-  close_in ic; s
+  close_in ic; Cstruct.of_string s
```

Comment:
I don't entirely know why/what is attempted here (esp. how often you expected `read_urandom` to be called in the `bench` test, and what kind of random you need), but maybe https://github.com/mirage/mirage-random-test would be worth to use here?

Reply:
This generates two key pairs that will be used in the benchmark. I don't think it's necessary to tie this part to mirage interfaces but thanks for the pointer!

Reply:
FWIW, the point I attempted to make:
- opening and reading from `/dev/urandom` is certainly not easy to port to some targets (i.e. win32, javascript, mirageos)
- do you need fast and good random? or is slow and bad random sufficient?
- you could use OCaml's `Random` module directly (from your response I read that you just need some random bytes, they don't need to be cryptographically secure)